frontend/packages/core/src/button.stories.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import type { Meta } from "@storybook/react";
+
+import type { ButtonProps } from './button'
+
+import { Button } from './button';
+
+export default {
+  title: 'Core/Button/Button',
```

Comment:
should this be 
```suggestion
  title: 'Core/Button',
```

not familiar with significance of titles in Storybook

Reply:
will update! initially thought we would want a button subfolder for the diff buttons (button, button group, clipboard button)